base.json

```diff
@@ -63,11 +63,14 @@
       "groupName": "prettier packages"
     },
     {
+      "description": "Update references to Docker tags for renovate/renovate in Markdown files",
       "packageNames": ["renovate/renovate"],
       "paths": [".md"],
-      "extends": ["schedule:weekly"],
+      "extends": ["schedule:weekly", ":semanticCommitScope(docs)"],
```

Comment:
If we want it to be `docs:` then shouldn't it be type=docs and scope=null ?

Reply:
```suggestion
      "extends": ["schedule:weekly"],
```

right 🤦‍♂️